src/utils/httpClient.ts

```diff
@@ -9,6 +9,13 @@ export interface ErrorResponse {
   body: any | undefined;
 }
 
+export interface PageMeta {
+  page: number;
+  pages: number;
+  page_size: number;
+  records: number;
+}
```

Comment:
Should we rework a little bit the naming for this interface? 🤔 
I see several reasons:
- NextJs already have a concept called "Page" `src/pages`, which creates ambiguity when 2 different concepts have the same name.
- this interface does not match with the usage in `src/store/index.tsx`:
![image](https://user-images.githubusercontent.com/77609814/173998744-fb1acfd8-e609-4a8b-b7e7-66ef9333ad0d.png)

My recommendation: using an adapter for the "PageMeta" object coming from the API. Using a single interface for everywhere in the React application. This local interface could have a different name (to prevent confusion with NextJs pages), such as:
```
interface BatchInfo {
  batch: number;
  totalBatches: number;
  batchRecords: number;
  totalRecords: number;
  }
  ```

Then, the adapter function could help you to "map" the JSON format to the `BatchInfo` format.

Feel free to suggest another implementation, the main idea is to find a way to reduce the confusion between NextJs Page and the API PageMeta :)

Reply:
Updated as you suggested here [1b29889](https://github.com/hanam1ni/react-surveyor/pull/72/commits/1b29889437c6d7d47ae4e1211350a81ef4968da6) with one change. For the `batchRecords`, I prefer to name it `batchSize` as it looks easier to understand to me

Reply:
Totally agree, `batchSize` is definitely better!!! :)